tools/wpt/expectation.json

```diff
@@ -1088,14 +1087,23 @@
         "Request interface: new Request('about:blank') must inherit property \"keepalive\" with the proper type",
         "Request interface: new Request('about:blank') must inherit property \"isReloadNavigation\" with the proper type",
         "Request interface: new Request('about:blank') must inherit property \"isHistoryNavigation\" with the proper type",
-        "Request interface: new Request('about:blank') must inherit property \"signal\" with the proper type",
         "Response interface: operation error()",
         "Response interface: operation redirect(USVString, optional unsigned short)",
         "Response interface: attribute body",
         "Response interface: attribute bodyUsed",
         "Response interface: calling redirect(USVString, optional unsigned short) on new Response() with too few arguments must throw TypeError",
         "Window interface: operation fetch(RequestInfo, optional RequestInit)"
-      ]
+      ],
+      "abort": {
+        "cache.https.any.html": false,
+        "general.any.html": [
+          "response.arrayBuffer() rejects if already aborted",
+          "response.blob() rejects if already aborted",
+          "response.formData() rejects if already aborted",
+          "response.json() rejects if already aborted",
+          "response.text() rejects if already aborted"
```

Comment:
These tests are not easy to solve without some significant refactors, so I will leave them failing for now. They are timing related (the promise resolves one microtask too late), and very unlikely to be noticed by a user.

Reply:
Is this related to `fetch` implementation or is it more general problem (like wrong timing of ops)?

Reply:
I am not quite sure yet